apps/prairielearn/src/tests/sharedQuestionPreview.test.ts

```diff
@@ -82,11 +81,32 @@ describe('Shared Question Preview', function () {
       isStudentPage: false,
     };
 
-    testQuestionPreviews(previewPageInfo, addNumbers, addVectors);
-
-    testFileDownloads(previewPageInfo, downloadFile, false);
-
-    testElementClientFiles(previewPageInfo, customElement);
+    describe('When questions are share_source_publicly but not shared_publicly', async function () {
```

Comment:
Ditto, not async. Same applies to the other `describe(...)` block below.

Reply:
fixed